server/events/markdown_renderer.go

```diff
@@ -262,8 +278,15 @@ func (m *MarkdownRenderer) renderProjectResults(results []command.ProjectResult,
 		tmpl = templates.Lookup("multiProjectVersion")
 	case common.Command == importCommandTitle:
 		tmpl = templates.Lookup("multiProjectImport")
+	case common.Command == stateCommandTitle:
```

Comment:
Instead of this, could we check if `len(resultsTmplData) == 1` and use the prefix `single` and if its `len(resultsTmplData) > 1`, use the prefix `multi` ?

Reply:
Ah interesting.

Reply:
Thank you for considering